<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>603</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>编辑规则</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ruleTitleLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>点击输入规则名称</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="contentTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>751</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
     </font>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>300</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>最小/最大值*</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>对应等级</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
     </font>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>307</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="creatButton">
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
        </font>
       </property>
       <property name="text">
        <string>新建</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
        </font>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modeComboBox">
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>较小值优先</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>较大值优先</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>点击【新建】按钮新建一个分数段
点击【删除】按钮删除选择的分数段
优先模式-----------------------
最小值优先：分数越小时，划分等级越高
此时应在【最小值】中填写每个分数段段的最小值（包括最小值）
最后一个最小值填写极端值（一般是0）和最高等级
较大值优先：分数越高，划分等级越高
此时应在【最大值】中填写每个分数段的最大值（包括最大值）
此时应在最后一个分数段的中填写一个很大的值（正常分数无法达到）和最高得分</string>
    </property>
    <property name="text">
     <string>鼠标悬停在此处查看提示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
